src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -1450,6 +1450,10 @@ private function registerValidationConfiguration(array $config, ContainerBuilder
             ->setArgument(2, $config['not_compromised_password']['enabled'])
             ->setArgument(3, $config['not_compromised_password']['endpoint'])
         ;
+
+        if (!class_exists(ExpressionLanguage::class)) {
```

Comment:
I don't know how to use the new function but wouldn't something like this the better way now?
```suggestion
        if (ContainerBuilder::willBeAvailable('symfony/expression-language', ExpressionLanguage::class, ['symfony/framework-bundle', 'symfony/validator'])) {
```

Reply:
I think that's good as is, validation services should be loaded in dev or no-dev.